deployment/scripts/postinst

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -e
+
+DATA_PATH=/var/lib/umwelt-info
+
+if [ -e "$DATA_PATH/index" ]
```

Comment:
This is not really a complete solution as it does not create the `umwelt-info` user nor the `/var/lib/umwelt-info` directory owned by it.

Maybe we should create both via

```console
> useradd --home-dir /var/lib/umwelt-info --create-home umwelt-info
```

if the directory is missing? And then create (and chown it to `umwelt-info:umwelt-info`) an empty `datasets` folder so the indexer can run and the server can start properly?

Reply:
Added that. Will need to test this tomorrow by removing the existing installation...

Reply:
Hopefully found a much simpler solution noticing that the indexer will always create an index even if reading the datasets fails. This means the server will come up with a missing datasets (and thereby an empty index) even though it wont serve anything until the harvester has run.